src/main/java/com/spartronics4915/frc2020/subsystems/Indexer.java

```diff
@@ -0,0 +1,71 @@
+package com.spartronics4915.frc2020.subsystems;
+
+import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+
+public class Indexer extends SpartronicsSubsystem {
+
+    public int mSpinnerId = -1;
```

Comment:
Remember to add all devices IDs to the [2020 control map](https://docs.google.com/spreadsheets/d/14O3Dj1Bl6tqiHFrCJ7wyVRDOqS5EZ11p4q9j2rYS-3Q/edit#gid=0).